tm2py/components/network/highway/highway_network.py

```diff
@@ -206,22 +205,24 @@ def _set_tolls(self, network: EmmeNetwork, time_period: str):
                         indent=True,
                     )
                     continue  # tolls will remain at zero
-                for src_veh, dst_veh in zip(src_veh_groups, dst_veh_groups):
-                    link[f"@bridgetoll_{dst_veh}"] = (
-                        float(data_row[f"toll{time_period.lower()}_{src_veh}"]) * 100
-                    )
-            # set valuetoll
-            elif link["@tollbooth"] >= valuetoll_start_tollbooth_code:
-                if run_dynamic_toll: # initialize valuetoll to 0
-                    for src_veh, dst_veh in zip(src_veh_groups, dst_veh_groups):
-                        link[f"@valuetoll_{dst_veh}"] = 0
-                else: # if not running dynamic toll, set per-mile valuetoll based on lookup
+                # set bridgetoll
+                if link["@tollbooth"] < valuetoll_start_tollbooth_code:
                     for src_veh, dst_veh in zip(src_veh_groups, dst_veh_groups):
-                        link[f"@valuetoll_{dst_veh}"] = (
-                            float(data_row[f"toll{time_period.lower()}_{src_veh}"])
-                            * link.length
-                            * 100
+                        link[f"@bridgetoll_{dst_veh}"] = (
+                            float(data_row[f"toll{time_period.lower()}_{src_veh}"]) * 100
                         )
+                # set valuetoll
+                elif link["@tollbooth"] >= valuetoll_start_tollbooth_code:
```

Comment:
```suggestion
                else:
```
I personally avoid using elif without an else in the end.

Reply:
thanks, updated in `5a64326`